internal/provider/bigquery_datamart_definition_resource.go

```diff
@@ -418,79 +418,14 @@ func (r *bigqueryDatamartDefinitionResource) Create(ctx context.Context, req res
 	if !plan.ResourceGroupID.IsNull() {
 		input.SetResourceGroupID(plan.ResourceGroupID.ValueInt64())
 	}
-	if plan.CustomVariableSettings != nil {
```

Comment:
Could you reorder each `Set*` method to match the sequence in the `Update` method? Please place `Schedules`, `Notifications`, and `Labels` after `DatamartBigqueryOption` for consistency. This also makes the diffs easier to read.

Reply:
thank you for reviewing!
I fixed it 
https://github.com/trocco-io/terraform-provider-trocco/pull/4/commits/00bf43991a5e7fc0891e4eff610d2bc352aa60b4
https://github.com/trocco-io/terraform-provider-trocco/pull/4/commits/7c99b4d6971c526b2cd77aedc6a956129dc8c036